assert.js:386
    throw err;
    ^
AssertionError [ERR_ASSERTION]: Graph does not have node A
    at PatternUserAttempt.addTimestamps \
 (/u/stjs/build-manager/add-timestamps.js:21:7)
    at PatternUserAttempt.buildGraph \
 (/u/stjs/build-manager/add-timestamps.js:15:10)
    at PatternUserAttempt.buildGraph \
 (/u/stjs/build-manager/variable-expander.js:5:11)
    at PatternUserAttempt.buildGraph \
 (/u/stjs/build-manager/pattern-user-attempt.js:5:11)
    at PatternUserAttempt.build \
 (/u/stjs/build-manager/skeleton-builder.js:10:10)
    at Object.<anonymous> (/u/stjs/build-manager/driver.js:3:9)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js \
 (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
